welcome/index.adoc

```diff
@@ -64,275 +64,4 @@ endif::[]
 
 ifdef::openshift-online,openshift-aro[]
 Start with **xref:../architecture/architecture.adoc#architecture-overview-architecture[Architecture]**.
```

Comment:
This line might be a bit lost outside the bullet points. Maybe add it as a bulleted point as a recommendation/consideration?

Reply:
I just removed it. Now that I really think about it, I think if we tell people to use the navigation then that is all they need. Also, more than likely people will be coming to docs for a specific thing and not to read top to bottom.